src/main/java/com/amazon/opendistroforelasticsearch/security/auditlog/impl/AbstractAuditLog.java

```diff
@@ -80,33 +79,16 @@
 import com.google.common.io.BaseEncoding;
 
 public abstract class AbstractAuditLog implements AuditLog {
-
-    private static final List<String> DEFAULT_DISABLED_CATEGORIES = Arrays.asList(AuditCategory.AUTHENTICATED.toString(), AuditCategory.GRANTED_PRIVILEGES.toString());
-
     protected final Logger log = LogManager.getLogger(this.getClass());
```

Comment:
I had renamed this to logger. @vrozov wanted it renamed back to log

Reply:
@debjanibnrj It is more consistent with how `Logger` is named in other places and avoids unnecessary diffs.

Reply:
@vrozov Fair point with reducing diffs but `auditConfigFilter.log(log)` would be confusing to anyone especially because this is around audit "logging"

Reply:
IMO, it is a problem with method name, not `Logger` name. Rename `AuditConfig.Filter.log()`.

Reply:
It is not a static function if thats what you mean.

Reply:
I mean that the minor readability problem is caused by `AuditConfig.Filter` class method name `log()`. It is not important whether it is static or not.

Reply:
If you want it renamed to something specific I will rename as you had previously suggested to use log() otherwise let's leave this. I m fine with this.

Reply:
I am fine the current function name or with a different function name if you want to add readability. I strongly prefer `log` not to be renamed to `logger`.

Reply:
@vrozov Can you approve this PR in that case ?